package.xml

```diff
@@ -38,7 +38,7 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Maybe the more correct thing to do here would be to remove the line (rather than replace `catkin` with `ament_cmake`). I think the only reason it is upstream is because `catkin_pkg` in ROS 1 is rather militant about all packages having a `buildtool_depend` on `catkin`. This package doesn't actually need `ament_cmake` to build.

However, leaving it `catkin` will also work. It would not affect the ros 2 build from source, though it would be an issue when trying to release it on the ROS 2 build farm.